mesonbuild/coredata.py

```diff
@@ -572,12 +572,11 @@ def set_options(self, opts_to_set: T.Dict[OptionKey, T.Any], subproject: str = '
         # Set prefix first because it's needed to sanitize other options
         pfk = OptionKey('prefix')
         if pfk in opts_to_set:
-            prefix = self.sanitize_prefix(opts_to_set[pfk])
+            prefix = self.optstore.sanitize_prefix(opts_to_set[pfk])
```

Comment:
Could have been caught by mypy, but coredata.py is unchecked. :D

Reply:
Indeed, and I had a PR with full typechecking done before the option refactor...